src/lib/LR1121Driver/LR1121.cpp

```diff
@@ -708,40 +767,44 @@ void ICACHE_RAM_ATTR LR1121Driver::GetLastPacketStats()
 
 void ICACHE_RAM_ATTR LR1121Driver::IsrCallback_1()
 {
-    instance->IsrCallback(SX12XX_Radio_1);  
+#ifndef UNIT_TEST
+    if (digitalRead(GPIO_PIN_DIO1))
+#endif
```

Comment:
Rather that having the `#ifndef` just add `LR1121Driver` to the `lib_ignore` line in `platformio.ini`